src/winetricks

```diff
@@ -6682,7 +6685,9 @@ load_dotnet45()
     w_call remove_mono
 
     # See https://appdb.winehq.org/objectManager.php?sClass=version&iId=25478 for Focht's recipe
-    w_call dotnet35
+
+    # Seems unneeded in wine-2.0
```

Comment:
That bug refers to allowing .msu installers to work. That's unrelated to if .net45 itself can install.

I've never used staging on a regular basis, but I can tell you that wine-2.4-55-g5b4f8a3 + this version of winetricks allows dotnet45 to install (with a 32-bit prefix, previously also tested with win64).